jax/_src/config.py

```diff
@@ -1449,6 +1449,57 @@ def _update_disable_jit_thread_local(val):
     update_thread_local_hook=lambda val: \
       update_thread_local_jit_state(default_matmul_precision=val))
 
+_logging_level_handler_set: dict[str, tuple[logging.Handler, int]] = {}
+
+def _update_logging_level_global(logging_level: str | None):
+  # remove previous handlers
```

Comment:
Can you move basically all of this logic to logging_config.py? I think you can also maybe combine some of it with existing logic there (or maybe not worth it since it's only a few lines of shared code)